maps/nerva/datums/nerva_jobs.dm

```diff
@@ -446,3 +446,20 @@
 
 /datum/job/submap
 	required_language = null
+
+//passenger. this has to be its own role for its own spawns.
+
+/datum/job/passenger
+	title = "Passenger"
+	department = "Civilian"
+	supervisors = "the Captain's tolerance"
+	department_flag = CIV
+	total_positions = -1
+	spawn_positions = -1
+	economic_power = 2
+	outfit_type = /singleton/hierarchy/outfit/job/nerva/passenger
+
+/datum/job/passenger/equip(mob/living/carbon/human/H)
+	if (H?.mind?.assigned_role == "Passenger")
+		H.mind.manual_pay_rate = -PASSENGER_FEE
+	. = ..()
```

Comment:
This assigned role check is redundant. This also feels a _little_ jank to me, an alternative solution could be to check for the assigned role in `/datum/mind::get_pay()` and return early there.
```suggestion
/datum/job/passenger/equip(mob/living/carbon/human/H)
	. = ..()
	H.mind?.manual_pay_rate = -PASSENGER_FEE
```